app/config/AppConfig.scala

```diff
@@ -45,6 +43,8 @@ class AppConfig @Inject() (config: Configuration, servicesConfig: ServicesConfig
     val feedbackFrontend: String = platformHost.getOrElse(config.get[String]("baseUrl.feedback-frontend"))
     val gg: String = config.get[String]("baseUrl.gg")
     val businessTaxAccountFrontend: String = platformHost.getOrElse(config.get[String]("baseUrl.business-tax-account-frontend"))
+    val timeToPayUrl: String = s"${servicesConfig.baseUrl("time-to-pay")}"
```

Comment:
small thing, but `s"${x}"` is equivalent to just `x` - this and the `val` below can be cleaned up a tiny bit

Reply:
lol sorry forgot to do that, will update

Reply:
Done 😄